app/web/features/markdown/MarkdownPage.tsx

```diff
@@ -183,7 +183,7 @@ export default function MarkdownPage({
           ))}
         </StyledBreadcrumbs>
         {!frontmatter.hide_title && (
-          <StyledTitle>{frontmatter.title}</StyledTitle>
+          <StyledTitle component="h1">{frontmatter.title}</StyledTitle>
```

Comment:
@nabramow do you know why it wouldn't be happy with this?

Reply:
`variant` tells you the style, `component` tells you the actual HTML tag to use, right? I want the appearance (variant) to be as it, but component to be `h1` so it's not just a huge styled `<p>`